ui/src/components/SampleView/ContextMenu.jsx

```diff
@@ -538,10 +488,28 @@ export default class ContextMenu extends React.Component {
     } else {
       optionList = menuOptions.NONE.map(this.listOptions);
     }
-    return (
-      <Dropdown.Menu show id="contextMenu" role="menu">
-        {optionList}
-      </Dropdown.Menu>
+
+    return createPortal(
```

Comment:
Note the use of [`createPortal(..., document.body)`](https://react.dev/reference/react-dom/createPortal) as mentioned.